examples/cool_features_demo.py

```
@@ -0,0 +1,319 @@
+async def demo_transformers():
+    """Demonstrate field transformers."""
+    print("=== 🔄 Field Transformers Demo ===")
+    
+    # Test date transformation
+    person1 = Person(
+        name="  john doe  ",  # Will be title-cased
+        birth_date="1990-05-15",  # String will be converted to date
+        email="JOHN.DOE@EXAMPLE.COM"  # Will be lowercased
+    )
+    
+    print(f"Original input: name='  john doe  ', birth_date='1990-05-15', email='JOHN.DOE@EXAMPLE.COM'")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)